src/components/device/ha-devices-picker.ts

```diff
@@ -3,7 +3,7 @@ import { customElement, property } from "lit/decorators";
 import { fireEvent } from "../../common/dom/fire_event";
 import { PolymerChangedEvent } from "../../polymer-types";
 import { HomeAssistant } from "../../types";
-import "./ha-device-picker";
+import { HaDevicePickerDeviceFilterFunc } from "./ha-device-picker";
```

Comment:
Still need the other import

Reply:
I then get an `eslint` error regarding duplicate imports. I don't have them in `ha-entities-picker.ts` though.